__tests__/app.test.js

```diff
@@ -97,3 +97,41 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .then((response) => {
+        const comments = response.body.comments; //array of comments
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments.length).toBe(2);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
```

Comment:
You can be more specific here.What article id do you expect these comments to have?

Reply:
To add one more thing to this, you could be testing that the typeof the property values is what you expect it to be! Right now you are only testing that your comments contain the property keys